### ISSUE TITLE
git@github.com:bytebin/deepworld-shared.git repository not found

### ISSUE THREAD

**Issue by ZufengW** [bytebin/deepworld-gameserver#1]
The `git@github.com:bytebin/deepworld-shared.git` repo referenced in the Gemfiles is not accessible.

**Comment by jsonperl** [comment]
Yep that had a few minor functions but one of them was acting as the vault for passwords and such, so we’re not opening that up. Is there something in particular you’re looking for?

**Comment by ZufengW** [comment]
I just wanted to install dependencies and run something (e.g. the tests). I tried commenting out the `deepworld-shared` line and skipping that but I had issues with some of the other dependencies. Maybe my version of Ruby (2.5.0) is too new.

**Comment by jsonperl** [comment]
I'll try and shim the shared project and update ruby at some point so the specs can run. No guarantees when though!

**Comment by LihrhaWolfe** [comment]
trying to run the server to show some friends this game, even though it is shut down, i still wanted to show it to them so i am trying to run the server files.  i keep getting stuck at this part though, any way around it?

edit; yes i know, necropost, but still need to know.

**Comment by jsonperl** [comment]
> trying to run the server to show some friends this game, even though it is shut down, i still wanted to show it to them so i am trying to run the server files. i keep getting stuck at this part though, any way around it?
> 
> edit; yes i know, necropost, but still need to know.

We haven't done it yet, sorry. But it's not gonna quite get ya there, you'd need a bunch of other subsystems that aren't that interesting to open-source or easy to run. And a dev build of the game that could connect to a local setup. We appreciate the interest!

**Comment by kuroppoi** [comment]
Sorry to dig this up again, but there's a few things that I am unable to find.
The Game Configuration appears to be incomplete; some fields from the items config are missing (such as layers and titles.)
Adding these isn't much of a problem, but it opens up the possibility of other (perhaps more important) things to be missing as well.
I also haven't been able to find any code responsible for generating new zones (I'd guess it was a seperate program taking care of that,) and that is something I am quite interested in.
